.gitignore

```diff
@@ -16,4 +17,4 @@ junit.xml
 .DS_Store
 
 ## Only track CI's lock
-yarn.lock
\ No newline at end of file
+yarn.lock
```

Comment:
O_O

What am I missing? I feel like the `yarn.lock` should be checked in?

Reply:
No idea, didn't do this on purpose haha. Reverting.